src/Sylius/Bundle/AdminBundle/Resources/config/routing.yml

```diff
@@ -101,3 +101,6 @@ sylius_admin_tax_rate:
 
 sylius_admin_zone:
     resource: "@SyliusAdminBundle/Resources/config/routing/zone.yml"
+
+sylius_admin_shipment:
```

Comment:
I would keep an alphabetical order and place it before `sylius_admin_shipping_category`

Reply:
Still relevant